queue/driver.go

```diff
@@ -0,0 +1,25 @@
+package queue
+
+import (
+	"github.com/goravel/framework/contracts/queue"
+)
+
+const DriverSync string = "sync"
+const DriverASync string = "async"
```

Comment:
The `async` driver means to save the queue in memory, right?

Reply:
> > The `async` driver means to save the queue in memory, right?
> 
> Consider using `go func` directly to start a goroutine running job.

It seems that using go func cannot return the job error because it is in a goroutine. Maybe we should return to your suggestion.